services/web/client/source/class/osparc/auth/LoginPage.js

```diff
@@ -121,8 +121,9 @@ qx.Class.define("osparc.auth.LoginPage", {
       });
 
       const platformVersion = osparc.utils.LibVersions.getPlatformVersion();
+      const platformName = osparc.utils.LibVersions.getPlatformName();
       if (platformVersion) {
-        const text = platformVersion.name + " v" + platformVersion.version;
+        const text = platformVersion.name + " " + platformVersion.version + (platformName.length ? ` (${platformName})` : " (production)");
```

Comment:
why defaulting to 'production' in that case? shouldn't it be 'undefined' or 'dev'?

Reply:
Well because the name is set in ``services/web/client/source/class/osparc/utils/LibVersions.js`` and for production, I am giving an empty string as the name because in a production environment, I don't want to show any string by the oSPARC logo.
In the case of the home page, we can show this (production) text close to the commit hash.